content/03-reference/01-tools-and-interfaces/07-preview-features/001-mssql/index.mdx

```diff
@@ -0,0 +1,88 @@
+
+---
+
+<TopBlock>
+
+The Microsoft SQL Connector is available as a preview feature in 2.11.0.
+
+</TopBlock>
+
+## Enable MSSQL preview feature
+
+To enable the MSSQL preview feature:
+
+1. Update the `client` and `datasource` blocks in your schema as shown:
+
+    ```prisma file=prisma/schema.prisma
+    generator client {
+        provider        = "prisma-client-js"
+    |    previewFeatures = ["microsoftSqlServer"]
+    }
+
+    datasource db {
+    |    provider = "sqlserver"
+        url      = env("DATABASE_URL")
+    }
+    ```
+
+1. Generate Prisma Client:
+
+    ```terminal
+    npx prisma generate
+    ```
+
+Alternatively, follow the MSSQL tutorial.
+
+## Type mapping
+
+| **Prisma type** | **MSSQL types** |
+|-----------------|-----------------|
+| `String`        | `char`<br />`varchar`<br />`nvarchar`<br />`text`<br />`ntext` |
+| `Float`         | `float(n)`<br />`number(p,s)`<br />`decimal(p,s)`<br />`money`<br />`smallmoney`<br />`real` |
+| `DateTime`      | `time`<br />`date`<br />`datetime`<br />`datetime2`<br />`smalldatetime`<br />`datetimeoffset` |
+| `Boolean`       | `bit`
+| `Integer`       | `tinyint`<br /> `smallint`<br />`int`<br />`bigint`
+
+## Supported versions
+
+See [Supported databases](../../../../more/supported-databases#preview).
+
+## Limitations and known issues
+
+### TLS required
+
+Prisma Client requires TLS to connect to an MSSQL database. Be aware that Apple are very specific about [what kind of TLS server certificates are accepted](https://support.apple.com/en-ca/HT210176), which means that MacOS users may not be able to connect to a SQL Server instance that uses a weak TLS certificate.
+
+To make local development easier on MacOS, you can add `encrypt=DANGER_PLAINTEXT` to the end of your connection string.
+
+> **Note**: `encrypt=DANGER_PLAINTEXT` must not be used in production.
+
+### Migration not fully supported
+
+Follow issue [#831 on GitHub](https://github.com/prisma/prisma-engines/issues/813) for more information.
+
+### Extended native types support not available
+
+Follow issue [#1183 on GitHub](https://github.com/prisma/prisma-engines/issues/1183) for more information.
+
+### Data model limitations
+
+#### Cannot use column with `UNIQUE` constraint and filtered index as foreign key
+
+MSSQL [only allows one `NULL` value in a column that has a `UNIQUE` constraint](https://docs.microsoft.com/en-us/sql/relational-databases/tables/unique-constraints-and-check-constraints?view=sql-server-ver15#Unique). For example:
+
+* A table of users has a column named `license_number`
+* The `license_number` field has a `UNIQUE` constraint
+* The `license_number` field only allows **one** `NULL` value
+
+The standard way to get around this issue is to create a filtered unique index that excludes `NULL` values. This allows you to insert multiple `NULL` values.
+
+*However*, this setup makes it impossible to use `license_number` as a foreign key in the database.
```

Comment:
We could possibly say what this means in Prisma terms, but it's not absolutely necessary :)

Reply:
Added that you'll get an error if you try to insert multiple uniques without the index